mixin/alerts/sidecar.libsonnet

```diff
@@ -22,6 +22,20 @@
               severity: 'critical',
             },
           },
+          {
+            alert: 'ThanosSidecarBucketOperationsFailed',
+            annotations: {
+              description: 'Thanos Sidecar {{$labels.job}} {{$labels.pod}} bucket operations are failing',
+              summary: 'Thanos Sidecar cannot connect to Prometheus',
+            },
+            expr: |||
+              rate(thanos_objstore_bucket_operation_failures_total{name="prometheus"}[5m]) > 0
```

Comment:
```suggestion
              rate(thanos_objstore_bucket_operation_failures_total{name="prometheus"}[5m]) > 0
```
Selectors should be templated, check out the other alert examples.

Reply:
right done

Reply:
AFAIT from the CI failures you need to run `make examples`

Reply:
ah thx!

Reply:
should be done